run_dir/design/project_samples.html

```diff
@@ -286,6 +287,17 @@ <h4 class="panel-title"><a data-toggle="collapse" data-parent="#undefined_fields
         </div>
       </form>
       <hr>
+      <!-- Woksets -->
+      <label for="tab_samples_worksets">Worksets:</label>
+      {% for workset in worksets %}
+        <div id="tab_samples_worksets" class="btn-toolbar" role="toolbar" aria-label="...">
```

Comment:
Need to remove `aria-label="..."` (this is for disabled people with screen readers, for help text)